ex-36/index.js

```diff
@@ -0,0 +1,12 @@
+const checkNum = (...args) => {
```

Comment:
Функции следует в явном видет принимать три параметра.
```suggestion
const checkNum = (a,b,c) => {
```